tests/alive-tv/glbconstchk3.tgt.ll

```diff
@@ -0,0 +1,5 @@
+@g = constant i32 0
+define i32 @f() {
+  %x = load i32, i32* @g
```

Comment:
please add a failing test as well.

Reply:
tests/alive-tv/refinement/const-fail.src.ll contains the test for mismatch between initial values of constants in src and tgt.